src/deserialize_sequence.cc

```diff
@@ -190,22 +191,26 @@ void extract_time_of_last_execution(gul14::string_view extract, Step& step)
     step.set_time_of_last_execution(extract_time("time of last execution", extract));
 }
 
-void extract_timeout(gul14::string_view extract, Step& step)
+template<typename Component>
```

Comment:
Maybe this can be even simpler: Couldn't this be a free function `Timeout parse_timeout(string_view)`? Then we could simply call it wherever it is needed:
```cpp
step. set_timeout(parse_timeout("45"));
```